packages/sw-lib/test/browser-unit/sw-unit/caching-strategies.js

```diff
@@ -0,0 +1,44 @@
+importScripts('/node_modules/mocha/mocha.js');
+importScripts('/node_modules/chai/chai.js');
+importScripts('/node_modules/sw-testing-helpers/build/browser/mocha-utils.js');
+
+importScripts('/packages/sw-lib/build/sw-lib.min.js');
+
+/* global goog */
+
+const expect = self.chai.expect;
+self.chai.should();
+mocha.setup({
+  ui: 'bdd',
+  reporter: null,
+});
+
+describe('Test swlib.cacheOnly', function() {
+  it('should be accessible goog.swlib.cacheOnly', function() {
```

Comment:
We talked about why these functions aren't just using arrow syntax offline. I think your reason about Mocha breaking decorated `it` bindings etc if arrows are used was fine. I did see https://github.com/skozin/arrow-mocha in case we wanted to explore how to work around this in future.